tests/foreman/cli/test_satellitesync.py

```diff
@@ -1828,7 +1828,7 @@ def test_postive_export_import_cv_with_file_content(
 
     @pytest.mark.parametrize(
         'function_synced_rh_repo',
-        ['rhae2'],
+        ['rhs9'],
```

Comment:
Instead of using rhs9 which would change when its EOL, how about using `rhsclient9` repo here for ISS?